src/bundle/Resources/config/services/controllers.yaml

```diff
@@ -13,6 +13,8 @@ services:
 
     EzSystems\EzPlatformUserBundle\Controller\PasswordResetController:
         autowire: true
+        arguments:
+            $actionResultHandler: '@EzSystems\EzPlatformUser\ExceptionHandler\NullActionResultHandler'
```

Comment:
Is this argument ok here? Wasn't it supposed to be an interface insted, aliased to a concrete result handler?

And maybe make it optional and initialize to `NullActionResultHandler` if it's missing, if we're not sure that the interface alias is registered?

Reply:
I have removed this altogether and put this on autowire shoulders.